main.tf

```diff
@@ -77,6 +77,10 @@ resource "aws_ecs_task_definition" "default" {
       }
     }
   }
+
```

Comment:
The task definition management will be moved into the prerender repo. By doing this the service controls the CPU, Memory, Docker Image and all settings related to its runtime. Terraform manages the rest of the AWS infrastructure. 

Until the Task Definition is moved into the prerender repo changes to for all settings besides the Docker Image will need to be done manually.